docs/editing.md

```diff
@@ -97,12 +97,17 @@ When a submission is ready to be accepted, we ask that the authors issue a new t
 Optionally you can ask EditorialBot to generate a checklist with all the post-review steps running the command: `@editorialbot create post-review checklist`
 
 Pre-publication steps:
+- (Optional) Run `@editorialbot create post-review checklist`
```

Comment:
This is a bit repetitive since it's written just above, but if someone goes too quickly to the list of steps, this suggestion can be easily missed.